Examples-Web/HoverButton/BasicHoverButton.js

```diff
@@ -0,0 +1,40 @@
+window.onload = function () {
+    // Set up TouchFree
+    TouchFree.Connection.ConnectionManager.init();
+    new TouchFree.InputControllers.WebInputController();
+    // Add basic inbuilt cursor
+    new TouchFree.Cursors.SVGCursor();
+
+    // Add eventListers to all the buttons
+    const buttons = document.getElementsByClassName("button");
+    for (let i = 0; i < buttons.length; i++) {
+        const element = buttons[i];
+        // When the cursor enters the button then call onHover
+        element.addEventListener("pointerenter", () => onHover(element));
+        // When the cursor leaves the button then call onHover
+        element.addEventListener("pointerleave", () => onDehover(element));
```

Comment:
Suggestion to change "dehover" to "unhover"? Dehover just hits my eye weirdly

Reply:
Updated